.circleci/config.yml

```diff
@@ -164,6 +164,11 @@ jobs:
           shell: /bin/bash
           command: ./Tests/scripts/slack_notifier.sh
           when: always
+      - run:
```

Comment:
Why is this step configured to run apart from all the other validations we have on the yml files?
I think it would make more sense to include this stage right after the Validate Files and Yaml stage

Reply:
See my comment: https://github.com/demisto/etc/issues/14971#issuecomment-491487511